INSTALL.adoc

```diff
@@ -25,7 +25,7 @@
 
         ./configure
 +
-This generates the three configuration files `config/Makefile`,
+This generates the three configuration files `config/Makefile.config`,
```

Comment:
Stray `config/` at the start of the path: ``This generates the three configuration files `Makefile.config`,``